components/benefit_expansion.js

```diff
@@ -72,12 +72,28 @@ export class BenefitExpansion extends Component {
       childBenefits,
       "family"
     );
+    const reduxState = this.props.reduxState;
+    const profileFilters = JSON.parse(
+      JSON.stringify(getProfileFilters(reduxState, this.props))
+    );
 
     let otherBenefits = t("benefits_b.eligible_open_veteran", {
       x: benefitName
     });
 
-    return (
+    return profileFilters["statusAndVitals"] === "deceased" ? (
```

Comment:
You don't need to use the `profileFilters` object here. You can just use `reduxState.statusAndVitals`. The `profileFilters` object just exists so we can loop over all the profile questions easily, but all the same data is in redux.